src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -92,65 +93,76 @@ public void doCommand() {
 
     @Override
     public void initialize() {
+        if(hardwareAvailable){
```

Comment:
consider adding a  private method with all this code.
that way, the code in this method is all at the same level of abstractoin